src/Components/WebView/Samples/PhotinoPlatform/testassets/PhotinoTestApp/Pages/TestPage.razor

```diff
@@ -0,0 +1,21 @@
+﻿@page "/"
+
+<div class="my-component">
```

Comment:
This is the test page used in the test. The test launches a Photino-based app and checks that the Counter button responds and updates the value in the `controlDiv`. This is exactly how we test things in the MAUI repo.